test/mvnormal.jl

```diff
@@ -1,6 +1,6 @@
 # Tests on Multivariate Normal distributions
 
-import PDMats: ScalMat, PDiagMat, PDMat
+import PDMats: ScalMat, PDiagMat, PDMat, PDSparseMat
```

Comment:
You'd have to put uses of `PDSparseMat` behind a `if isdefined(PDMats, :PDSparseMat)` since it's not defined when Julia is built without GPL libraries.